utils/sl/fake_scons/SCons/Script/__init__.py

```diff
@@ -343,6 +343,9 @@ def Glob(*args):
     """Fake Glob"""
     return []
 
+def Split(*args):
```

Comment:
(pylint-unused-argument) Unused argument 'args'

Reply:
I believe if you use _args, this one will go away